apiserver/service/bitrise_config.go

```diff
@@ -37,6 +59,12 @@ func GetBitriseYMLHandler(w http.ResponseWriter, r *http.Request) {
 
 // PostBitriseYMLHandler ...
 func PostBitriseYMLHandler(w http.ResponseWriter, r *http.Request) {
+	contStr, err := fileutil.ReadStringFromFile(config.BitriseYMLPath)
+	if err == nil && HasConfigVersionConflict(r, contStr) {
+		w.WriteHeader(http.StatusConflict)
```

Comment:
Maybe it's a bit more human friendly if we respond with an error message as well, we could create a shared method for that and use it in both handlers, seems like we have an "established" error message format already.